src/handlers/thirdparty.ts

```diff
@@ -34,4 +36,37 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
 
   let metric = buildCountFromThirdParty()
   metric.save()
-}
\ No newline at end of file
+}
+
+export function handleItemUpdated(event: ItemUpdated): void {
+  let item = Item.load(event.params._itemId)
+  if (item == null) {
+    return
+  }
+
+  item.metadata = event.params._metadata
+
+  item.save()
+}
+
+export function handleItemAdded(event: ItemAdded): void {
+  let collectionId = getCollectionId(event.params._itemId)
+  let itemId = getItemId(event.params._itemId)
+  if (collectionId == null || itemId == null) {
+    return
+  }
+
+  let item = Item.load(event.params._itemId)
+  if (item === null) {
+    item = new Item(event.params._itemId)
+  }
+
+  item.rawMetadata = event.params._metadata
+  item.thirdParty = event.params._thirdPartyId
+  item.id = event.params._itemId
+  item.isApproved = false
+  item.searchCollectionId = collectionId!
+  item.searchItemId = itemId!
```

Comment:
`metadata` is missing

Reply:
Build count for item is missing too. Check third parties

Reply:
Done!